.github/workflows/ci.yml

```diff
@@ -0,0 +1,23 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - '*'
+
+jobs:
+  macOS:
+    strategy:
+      matrix:
+        xcode:
+          - '13.2'
```

Comment:
Why should we use 13.2 and not latest?

Reply:
GitHub actions has some delay adding latest Xcode version, but this is just an example, you could bump the version whatever you like